show/main.py

```diff
@@ -1363,8 +1363,10 @@ def syslog(verbose):
         data = syslog_file.readlines()
     for line in data:
         if line.startswith("*.* @"):
-            line = line.split(":")
-            server = line[0][5:]
+            end = line.find("]")
```

Comment:
>find [](http://example.com/codeflow?start=23&length=4)

Better to use regex or something more smarter, in order to make parsing flexible to all the valid config:
1. @ipv4:port
2. @[ipv4]:port
3. @[ipv6]:port